.github/workflows/guardrails-validation.yaml

```diff
@@ -30,4 +30,4 @@ jobs:
         run: pwd && ls
       - name: nomos vet guardrails
         run: |
-          ls && docker run -v $PWD:/guardrails gcr.io/config-management-release/nomos:stable nomos vet --no-api-server-check --source-format unstructured --path /guardrails/configs
```

Comment:
the ls command before docker run - bothered me in the past - should have raised it but I was unable to test locally at the time.

LGTM